assets/javascripts/discourse/widgets/discourse-post-event.js

```diff
@@ -32,20 +32,13 @@ export default createWidget("discourse-post-event", {
   },
 
   showAllInvitees(params) {
-    const postId = params.postId;
-    const title = params.title || "title_invited";
-    const extraClass = params.extraClass || "invited";
-    const name = "discourse-post-event-invitees";
-
-    this.store.find("discourse-post-event-event", postId).then((eventModel) => {
-      showModal(name, {
-        model: eventModel,
-        title: `discourse_post_event.invitees_modal.${title}`,
-        modalClass: [`${dasherize(name).toLowerCase()}-modal`, extraClass].join(
-          " "
-        ),
+    this.store
+      .find("discourse-post-event-event", params.postId)
+      .then((eventModel) => {
+        this.modal.show(DiscoursePostEventInvitees, {
+          model: { event: eventModel, params },
```

Comment:
I'd pass `title` and `extraClass` rather than whole `params` (even just to avoid super deep `this.args.model.params.[prop]` in the component 😅)